isis/src/base/apps/isis2std/tsts/toobig/Makefile

```diff
@@ -1,14 +0,0 @@
-APPNAME = isis2std
-
-include $(ISISROOT)/make/isismake.tsts
-
-commands:
-	if [ `$(APPNAME) from=$(INPUT)/large.lbl \
-	  to=$(OUTPUT)/isis2stdTruth1.png \
-	  mode=grayscale \
-	  format=png \
-	  stretch=linear \
-	   >& $(OUTPUT)/error.txt` ]; \
-	then \
-	  true; \
-	fi;
```

Comment:
I don't think this is worth testing through the tests as it requires generating a file over 2 GB in size. So, this is the only test I didn't convert.